### ISSUE TITLE
slaves file requires blank line at end (otherwise last slave won't be started)

### ISSUE THREAD

**Issue by andrehacker** [stratosphere/stratosphere#93]
The conf/slaves file requires an empty/blank line at the end.
If there is no trailing empty line, the start-cluster.sh (and maybe other files) will not start the last slave.

This issue - and a possible workaround - is described [here](http://linux.derkeiler.com/Mailing-Lists/SuSE/2008-11/msg00196.html).

**Comment by rmetzger** [comment]
This needs to be reworked (its easy to do).

Our system should be able to handle empty lines in the slave file.
Currently, you have to exactly ONE newline at the end of the "slaves" file.
If you have two at the end, you'll get this exception

```
Cannot extract topology information from line ""
        at eu.stratosphere.nephele.topology.NetworkTopology.fromFile(NetworkTopology.java:68)
        at eu.stratosphere.nephele.instance.cluster.ClusterManager.loadNetworkTopology(ClusterManager.java:432)
        at eu.stratosphere.nephele.instance.cluster.ClusterManager.<init>(ClusterManager.java:316)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:39)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:27)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:513)
        at java.lang.Class.newInstance0(Class.java:355)
        at java.lang.Class.newInstance(Class.java:308)
        at eu.stratosphere.nephele.jobmanager.JobManagerUtils.loadInstanceManager(JobManagerUtils.java:128)
        at eu.stratosphere.nephele.jobmanager.JobManager.<init>(JobManager.java:234)
        at eu.stratosphere.nephele.jobmanager.JobManager.main(JobManager.java:388)

```

And this error from the `start-cluster.sh` script:

```
ssh: Could not resolve hostname nohup /bin/bash /share/hadoop/stratosphere-dev/stratosphere/bin/../bin/nephele-taskmanager.sh start : Name or service not known
```

**Comment by rmetzger** [comment]
@markus-h is assigned to this task.

**Comment by StephanEwen** [comment]
Fixed in bb5d7708d8ddd7791d61b7a3f145cdbfbefa6fc3
